lms/djangoapps/verify_student/views.py

```diff
@@ -427,6 +444,31 @@ def _redirect_if_necessary(
         if url is not None:
             return redirect(url)
 
+    def _get_verified_modes_for_course(self, course_key):
+        """Retrieve unexpired and expired verified modes for a course.
+
+        Arguments:
+            course_key (CourseKey): The location of the course.
+
+        Returns:
+            Tuple of `(verified_mode, expired_verified_mode)`.  If provided,
```

Comment:
That does make sense, but I think the change would be pretty involved.  Oddly enough, the `CourseMode` class methods don't return Django models; they return `namedtuple`s.  If they returned Django models, it would be easy to add an `is_expired` property, calculated from the expiration date and current time.  This is messier with named tuples, which don't support dynamic properties.